smoke/_init.sh

```
@@ -90,18 +89,51 @@ fail() {
 }
 
-tmp=$(mktemp -d)
-trap cleanup EXIT
-builddir=${1-}
-run_id=$(echo $SRANDOM$SRANDOM | base32 -w6 | tr '[:upper:]' '[:lower:]' | head -n1)
+tap_counter=0
+port_add() {
+	local name="$1"
+	local vlan="$2"
+	shift 2
+	local mac=$(echo "$name" | md5sum | sed -E 's/(..)(..)(..)(..)(..).*/02:\1:\2:\3:\4:\5/')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)